modules/ts/src/ts_perf.cpp

```diff
@@ -665,11 +669,13 @@ static int perf_validation_idle_delay_ms = 3000; // 3 sec
 static void loadPerfValidationResults(const std::string& fileName)
 {
     perf_validation_results.clear();
+
+    float value = 0;
+#ifndef WINRT
     std::ifstream infile(fileName.c_str());
```

Comment:
You definitelly should fix/emulate ifstream/ofstream for WINRT. This part of code is unmaintanable.